pkg/net/libp2p/key.go

```diff
@@ -63,10 +67,11 @@ func networkPublicKeyToOperatorPublicKey(
 ) (*operator.PublicKey, error) {
 	switch publicKey := networkPublicKey.(type) {
 	case *libp2pcrypto.Secp256k1PublicKey:
+		ecdsaPublicKey := (*secp.PublicKey)(publicKey).ToECDSA()
```

Comment:
I think we can convert to `btcec`:
```suggestion
		btcecPublicKey := (*btcec.PublicKey)(publicKey)
```